src/cairoWriter/writers/variableDeclarationStatementWriter.ts

```diff
@@ -68,17 +69,17 @@ export class VariableDeclarationStatementWriter extends CairoASTNodeWriter {
         return [writer.write(getDeclarationForId(id))];
       }
     });
+    const solTyp = safeGetNodeType(node.vInitialValue, this.ast.inference);
+    const cairoTyp = CairoType.fromSol(solTyp, this.ast);
+    let namePortion: string;
     if (declarations.length > 1) {
-      return [
-        endent`${documentation}
-          let (${declarations.map((decl) => `mut ${decl}`).join(', ')}) = ${writer.write(
-          node.vInitialValue,
-        )};`,
-      ];
+      namePortion = `(${declarations.map((decl) => `mut ${decl}`).join(', ')})`;
```

Comment:
You eliminated `documentation`, was this intentional?

Reply:
I didn't eliminate it, it's on [line 81](https://github.com/NethermindEth/warp/pull/1069/files#diff-19bf5bcf6e7092aceebe553503413f8678e36a90cccfd4e15768532ac00e2b33R81)